src/component/DailyNoteEditorView.svelte

```diff
@@ -47,19 +49,27 @@
     }
 
     async function createNewDailyNote() {
-        const fileFormat = getDailyNoteSettings().format || 'YYYY-MM-DD';
         const currentDate = moment();
         if (!hasCurrentDay) {
             const currentDailyNote: any = await createDailyNote(currentDate);
             renderedDailyNotes.push(currentDailyNote);
 
-            renderedDailyNotes = renderedDailyNotes.sort((a, b) => {
-                return parseInt(moment(b.basename, fileFormat).format('x')) - parseInt(moment(a.basename, fileFormat).format('x'));
-            });
+            renderedDailyNotes = sortDailyNotes(renderedDailyNotes);
             hasCurrentDay = true;
         }
     }
 
+    function startFillViewport() {
+        if (!intervalId) {
+            intervalId = setInterval(infiniteHandler, 100); 
```

Comment:
Seems to be that my main Vault is slower than one with smaller files. The choosen delay for `setInterval` seems to be too large for general usage. I tested it with 1ms and 10ms. Both worked the same for me with the small files Test-Vault. So, from my point of view it could be reduced to 1ms to allow proper filling of editor.

Reply:
@calvinwyoung could you give it a try if a 1ms delay fixes the issue for you?

[Obsidian-Daily-Notes-Editor_1ms_delay.zip](https://github.com/user-attachments/files/15973522/Obsidian-Daily-Notes-Editor_1ms_delay.zip)

Reply:
@questmaster Brilliant, it's only be one day but this seems to work reliably for me. Would it be worth packaging that change into the next version?

Reply:
```suggestion
            intervalId = setInterval(infiniteHandler, 1); 
```
@Quorafind , please fix this line.

Reply:
Thanks, I will take a look at this.